tools/codegen/gen_backend_stubs.py

```diff
@@ -300,17 +303,20 @@ def make_file_manager(install_dir: str) -> FileManager:
         # This could be useful if a backend wants to quickly set up a noop yaml file but doesn't have any kernels ready yet.
         return
 
-    class_name = backend_indices[backend_key].native_function_class_name()
+    if class_name is None:
```

Comment:
Can you add a detailed comment here similar to what you described in the PR? Just to make it clear that the `class_name` yaml argument is completely optional, but if specified it allows you to override the name of the class that all generated kernel definitions live under.

This bit of the code is probably the closest we have to documentation for the yaml, and we probably want to make it clear that the argument is totally optional (versus some other arguments that toggle important behavior).

Reply:
sure, done, thanks.